data/brands/shop/convenience.json

```diff
@@ -550,6 +550,25 @@
         "shop": "convenience"
       }
     },
+    {
+      "displayName": "Arambagh Foodmart",
+      "id": "null",
+      "locationSet": {"include": ["in"]},
+      "matchNames": ["arambaghs foodmart", "arambagh chicken"],
+      "tags": {
+        "brand": "Arambagh Foodmart",
+        "brand:wikidata": "Q130550508",
+        "name": "Arambagh Foodmart",
+        "name:en": "Arambagh Foodmart",
+        "name:bn": "আরামবাঘ ফুডমার্ট",
+        "shop": "convenience",
+        "air_conditioning": "yes",
+        "payment:cash": "yes",
+        "contact:website":"https://arambaghfoodmart.com/",
```

Comment:
These lines (and similar tags in the other entries) will be automatically removed by the build script if this PR is approved. Their values should be added to the brand's Wikidata page instead.

Reply:
Got it. Will do.